main.py

```diff
@@ -125,13 +125,18 @@ async def on_typing(channel, user, when):
     logging.debug('Typing from {0.name}'.format(user))
 
 @mbox.event
-async def on_guild_join(guild):
+async def on_guild_join(guild: discord.Guild):
     logging.info('Joined Server: {0.name}'.format(guild))
```

Comment:
```suggestion
    logging.info(f'Joined Server: {guild.name}')
```
As of Python 3.6, a better alternative to using String format() is to use f-strings.

https://stackoverflow.com/questions/42379271/format-function-alternative-in-python